dev-bot/botfile.js

```diff
@@ -0,0 +1,110 @@
+module.exports = {
+  /*
+    Where the content is stored
+    You can access this property from `bp.dataLocation`
+  */
+  dataDir: process.env.BOTPRESS_DATA_DIR || './data',
+
+  /*
+    The port on which the API and UI will be available
+   */
+  port: process.env.BOTPRESS_PORT || process.env.PORT || 3000,
+
+  /*
+    Some modules might generate static configuration files
+   */
+  modulesConfigDir: process.env.BOTPRESS_CONFIG_DIR || './modules_config',
+
+  /*
+    Path to Content Forms
+   */
+  formsDir: './forms',
+
+  /*
+    Path to Content Forms Data
+   */
+  formsDataDir: './forms_data',
+
+  /*
+    By default logs are enabled and available in `dataDir`
+   */
+  disableFileLogs: false,
+  log: {
+    file: 'bot.log',
+    maxSize: 1e6 // 1mb
+  },
+
+  dialogs: {
+    timeoutInterval: '15m',
+    janitorInterval: '10s'
+  },
+
+  /*
+    Botpress collects some anonymous usage statistics to help us put our efforts at the right place
+   */
+  optOutStats: false,
+
+  /*
+    Where the notifications are stored.
+    TODO: These should be stored in the database
+   */
+  notification: {
+    file: 'notifications.json',
+    maxLength: 50
+  },
+
+  /*
+    By default ghost content management is only activated in production
+   */
+  ghostContent: {
+    enabled: process.env.NODE_ENV === 'production' || process.env.BOTPRESS_GHOST_ENABLED
```

Comment:
I think we should reverse this, i.e. we can still disable it even if in production?

Reply:
You mean invert? Right now it's "any non-empty string in BOTPRESS_GHOST_ENABLED" means "enabled"
But you can change this logic in your specific bot's file to match your requirements, and it's unrelated to the specific dev bot.
Obviously do as you feel the best in the cli template.